server/src/test/java/org/opensearch/index/shard/ReleasableRetryableRefreshListenerTests.java

```diff
@@ -58,6 +58,17 @@ public void beforeRefresh() {}
             protected Logger getLogger() {
                 return logger;
             }
+
+            @Override
+            protected boolean shouldRunAsync() {
+                return false;
+            }
+
+            @Override
+            protected String getRemoteRefreshThreadPoolName() {
+                return ThreadPool.Names.REMOTE_REFRESH_SEGMENT_SYNC;
+            }
+
```

Comment:
if we have duplicate this a lot of times, can we extend the class and have a TestReleasableRetryableRefreshListener that just overrides the above methods and uses it everywhere in this test?